src/gui/guiScrollBar.h

```diff
@@ -67,6 +67,7 @@ class GUIScrollBar : public IGUIElement
 private:
 	void refreshControls();
 	s32 getPosFromMousePos(const core::position2di &p) const;
+	f32 interpolate_scroll(f32 from, f32 to, f32 amount);
```

Comment:
See style guidelines: https://dev.minetest.net/Code_style_guidelines
```suggestion
	f32 interpolateScroll(f32 from, f32 to, f32 amount);
```

Reply:
Hah, habit :)